ccp/lib/processors/pipeline_information/openshift.py

```diff
@@ -0,0 +1,48 @@
+"""
+This file contains basic processor to process OpenShift information
+"""
+
+from ccp.lib.processors.base import QueryProcessor
+from ccp.lib.clients.openshift.client import OpenShiftCmdClient
+
+
+class OpenShiftCommandProcessor(QueryProcessor):
+
+    def __init__(
+            self,
+            openshift_insecure=True,
+            openshift_ca_path=None,
+            openshift_client_cert_path=None,
+            openshift_client_key_path=None
+    ):
+        """
+        :param openshift_insecure: Default True: If true, then connection is
+        insecure even with https so certs verification is ignored
+        :type openshift_insecure: bool
+        :param openshift_ca_path: Default None: If set, this is used to verify
+        certs, assuming insecure is False
+        :type openshift_ca_path: str
+        :param openshift_client_cert_path: Default None: If provided then client
+        cert path is used.
+        :type openshift_client_cert_path: str
+        :param openshift_client_key_path: Default None: If provided, then client
+        key path is used.
+        :type openshift_client_key_path: str
+        """
+        self.oc_client = OpenShiftCmdClient(
+            insecure=openshift_insecure,
+            ca_path=openshift_ca_path,
+            client_cert_path=openshift_client_cert_path,
+            client_key_path=openshift_client_key_path
+        )
+
+    def get_projects(self):
+        """
+        Gets the list of OpenShift namespaces/projects
+        :return: List of namespaces
+        :rtype list
+        :raises Exception
+        :raises subprocess.CalledProcessError
+        :raises ccp.lib.exceptions.CommandOutputError
+        """
+        return self.oc_client.get_projects().split('\n')[-1]
```

Comment:
this will return only the last line of the result of `oc projects -q` ?

https://github.com/CentOS/container-pipeline-service/pull/650/files#diff-70cbc6ccc4f9579d7955ccdd538d0c95R172